lib/App/DuckPAN/Cmd/Test.pm

```diff
@@ -6,27 +6,44 @@ with qw( App::DuckPAN::Cmd );
 
 use MooX::Options protect_argv => 0;
 
+use Path::Tiny;
+
 option full => (
-	is          => 'ro',
-	lazy        => 1,
-	short       => 'f',
-	default     => sub { 0 },
-	doc         => 'run full test suite via dzil',
```

Comment:
Fair enough.

I've got to figure out how to get Vim to let me use tabs for specifically this project ^^ (Emacs should be easier).